scapy/utils.py

```diff
@@ -1876,11 +1888,13 @@ def _read_block_epb(self, block, size):
 
         process_information = {}
         for code, value in options.items():
-            if code in [0x8001, 0x8003]:  # PCAPNG_EPB_PIB_INDEX, PCAPNG_EPB_E_PIB_INDEX
+            # PCAPNG_EPB_PIB_INDEX, PCAPNG_EPB_E_PIB_INDEX
```

Comment:
Please revert changes from line 1891 to 1898 as they are not needed for your PR.

Reply:
It's fine, I prefer it.